toolkit/scripts/pkggen.mk

```diff
@@ -67,30 +75,17 @@ $(graph_file): $(specs_file) $(go-grapher)
 		$(logging_command) \
 		--output $@
 
+# We want to detect changes in the RPM cache, but we are not responsible for directly rebuilding any missing files.
+$(CACHED_RPMS_DIR)/%: ;
+
 # Remove any packages which don't need to be built, and flag any for rebuild if
 # their dependencies are updated.
 ifneq ($(CONFIG_FILE),)
 # If an optional config file is passed, validate it and any files it includes. The target should always depend
 # on the value of $(CONFIG_FILE) however, so keep $(depend_CONFIG_FILE) always.
```

Comment:
Is this comment still valid?

Reply:
I think so... Its a bit oddly worded (my fault) but we always want to pass the config file through the validator when we first use it. This optionally adds a dependency on that check if the CONFIG_FILE variable is set.